### PR TITLE
NHibernate supports only SQL Server and Oracle

### PR DESCRIPTION
**Pull request by MarcinHoppe**

### CODE REVIEW

**Comment by MarcinHoppe** [comment]
Fixes Particular/NServiceBus.NHibernate#268.

**Comment by MarcinHoppe** [comment]
@Particular/docs-maintainers This is ready for review (and maybe also merge).

File: nservicebus/nhibernate/index.md [hunk 1]
```diff
@@ -22,6 +22,15 @@ Uses the [NHibernate ORM](http://nhibernate.info/) for persistence.
  * [Outbox](/nservicebus/outbox/)
 
 
+## Supported database engines
+
+
+ * [SQL Server](https://www.microsoft.com/en-au/sql-server/) ([Version 2012](https://docs.microsoft.com/en-us/sql/release-notes/sql-server-2012-release-notes) and above).
```
Comment (DavidBoike) [inline nservicebus/nhibernate/index.md]: Perhaps make it Microsoft SQL Server to be explicit?

**Comment by MarcinHoppe** [comment]
Great catch @DavidBoike! Fixed.

**Comment by DavidBoike** [comment]
@MarcinHoppe Is the supported ODP.NET driver the same one we use for SQL Persistence, i.e. [this NuGet package](https://www.nuget.org/packages/Oracle.ManagedDataAccess/)?

If so maybe in [this section](https://github.com/Particular/docs.particular.net/blob/master/nservicebus/nhibernate/index.md#supported-database-engines) the package name should be linked to the NuGet package page.

**Comment by MarcinHoppe** [comment]
I believe so. Adding a link for clarity makes sense, a PR will follow shortly.

**Comment by MarcinHoppe** [comment]
@DavidBoike I created #2856.

**Pull request merged**